src/components/ImageDetails/CopyLicense.vue

```diff
@@ -113,9 +113,9 @@
         >
           {{ imageTitle }}
           <span v-if="image.creator"> by {{ image.creator }} </span>
-          is licensed under
+          is marked under
```

Comment:
I think you have to use `is marked under` for tools and `is licensed with` for licenses

Reply:
Yes @avats-dev, you need to check `isTool` here and use `marked under` for tools and `licensed with` for licenses.